docs/Reference/Routes.md

```diff
@@ -42,7 +42,7 @@ fastify.route(options)
   [here](./Validation-and-Serialization.md) for more info.
 
   * `body`: validates the body of the request if it is a POST, PUT, PATCH,
-    TRACE, or SEARCH method.
+    TRACE, SEARCH, PROPFIND, PROPPATCH, LOCK or UNLOCK method.
```

Comment:
And also not familiar with webdav but just pointing out that UNLOCK method wasn't added anywhere if that was unintentional

Reply:
After a check in the webdav RFC, UNLOCK method does not require a body, a mistake fixed in https://github.com/fastify/fastify/pull/5411/commits/54bf25c55c3c3fd7a53d581f01b3fa4077a4947e

I forgot to update the doc, sorry !

Reply:
done in https://github.com/fastify/fastify/pull/5411/commits/d59e5946a4f7f38fd6ec32fccd4b247efd3526a5